modules/core/src/core/animation-loop.js

```diff
@@ -211,6 +215,9 @@ export default class AnimationLoop {
     this.onRender(this.animationProps);
     // end callback
 
+    // clear needsRedraw flag
+    this._clearNeedsRedraw();
```

Comment:
Nit: use `getNeedsRedraw(clearRedrawFlag = false)` to align with the rest of this codebase?